hatchways/team-oak
# Feature--Backend--Request-Controller

### What this PR does (required):
- Adds controller for requests according to the following endpoints
    - GET /request/load - Gets list of requests for currently logged-in user
    - POST /request/new - Creates a new request and saves it in the database
    - PATCH /request/update - Updates provided boolean values for accepted, declined, and paid fields of Request object currently saved in database
- Adds helper functions in utils/ for potential use throughout the project

### Any information needed to test this feature (required):
- Add dummy user to database by running test "POST /request/new"
- Check for dummy user by running test "GET /request/load"

### Any issues with the current functionality (optional):
- The testing is incomplete
